spec/actions_spec.rb

```diff
@@ -232,12 +253,113 @@ def self.name
     end
 
     it 'deletes a non-existing record' do
-      instance = klass.new(value: 1)
+      instance = klass.new(value: 1, indexed_value: 1)
       expect(instance.id).to be_nil
       expect {
         instance.destroy
       }.to_not raise_error
       expect(klass.count).to eq 0
     end
   end
+
+  context 'custom indexes: ' do
+    let!(:time_now) { Time.zone.now}
+    let!(:instance) { klass.create!(indexed_value: 1, time_value: time_now) }
+    let!(:instance_2) { klass.create!(indexed_value: 2, time_value: nil) }
+
+    it 'creates an attribute in index content hash for custom index' do
+      unless ENV['REDCORD_SPEC_USE_CLUSTER'] == 'true'
+        index_string = klass.redis.hget("#{klass.model_key}:custom_index_first_content", instance.id)
+        expect(index_string).to_not be(nil)
+      end
```

Comment:
you can use `klass.redis.scan_each_shard("#{klass.model_key}:custom_index_first_content*") { |key| expect_key_not_empty }` 
so it works with both cluster mode on/off